sdk/cosmosdb/Microsoft.Azure.Management.CosmosDB/src/Microsoft.Azure.Management.CosmosDB.csproj

```diff
@@ -6,16 +6,16 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.Management.CosmosDB</PackageId>
     <Description>Provides developers with libraries for the CosmosDB under Azure Resource manager to manage CosmosDB Account, Databases and child resources and their available management capabilities. Create, Delete, Update CosmosDB Account, Databases and child resources. Note: This client library is for CosmosDB under Azure Resource Manager.</Description>
-    <Version>3.3.0</Version>
```

Comment:
This PR introduces some breaking changes, why only bump the minor version? And it also uses a preview API, we shouldn't use stable version.

Reply:
I am not introducing any breaking change in this PR. Just adding more apis and updating some api with new functionalities. Which version is should update to?